sleap/gui/widgets/docks.py

```diff
@@ -625,10 +630,52 @@ def create_video_unlink_button(self) -> QWidget:
             hb, "Unlink Video", main_window.commands.unlink_video_from_camera
         )
 
+        self.add_button(hb, "Add to Group", self._add_camera_to_group)
+
         hbw = QWidget()
         hbw.setLayout(hb)
         return hbw
 
+    def _add_camera_to_group(self):
+        """Add the selected camera to a group."""
+        camera = self.main_window.state.get("camera")
+        if not camera:
+            QMessageBox.information(
+                self.main_window,
+                "No Camera Selected",
+                "Please select a camera to add to a group."
+            )
+            return
+        
+        # Get available groups
+        if not self.main_window.state.get("camera_groups") or len(self.main_window.state["camera_groups"]) == 0:
```

Comment:
`GuiState` has a `__getitem__` method, so we at least won't run into an error if the key doesn't exist.
https://github.com/talmolab/sleap/blob/653144760882aa3f5c7f1199366d45aba0bccaed/sleap/gui/state.py#L52-L54

How about we use `get`, but with a default of an empty list:
```suggestion
        if len(self.main_window.state.get("camera_groups", []) == 0:
```

Reply:
ex:
```python
In [20]: from sleap.gui.state import GuiState

In [21]: state = GuiState()

In [22]: state[3]

In [23]: state.get(3, [])
Out[23]: []
```